### PR TITLE
Add chat cmd handler with default chat_id filtering

### LINKED ISSUE #257: Need a `CommandHandler` wrapper to provide `chat_id` for all skills and modes
Now to add skill or mode we need to call `CommandHandler` like: 
```python
  dp.add_handler(
        CommandHandler(
            "version",
            _version,
            filters=Filters.chat(username=get_group_chat_id().strip("@"))
            & admin_filter,
            run_async=True,
        ),
        version_handlers_group,
    )
```

The problem is here: `filters=Filters.chat(username=get_group_chat_id().strip("@"))`. We need to duplicate this like for all skill\mode add-commands. 

I'd like to make a `CommandHandler` which will set `filters` argument by default according to settings.

So, it should look like: 

```python
  dp.add_handler(
        ChatCommandHandler(
            "version",
            _version,
            run_async=True,
        ),
        version_handlers_group,
    )
```

We can extract `run_async=True` as well, because in 100% cases we'd like to run our command in `async` way. So, it could look like:
```python
  dp.add_handler(
        ChatCommandHandler("version", _version),
        version_handlers_group,
    )
```

### PR DESCRIPTION
**Pull request by egregors**
Closes #257 

- [x] Custom wrapper for `CommandHandler`
  - [x] support both chat_id (int) and chat_username (str)
  - [x] `chat_id` filtering by default
  - [x] `run_async=True` by default

### CODE REVIEW

**Review by getjump — APPROVED** [review]

**Comment by getjump** [comment]
LGTM!

File: bot/skills/__init__.py [hunk 1]
```diff
@@ -34,16 +38,61 @@
...
+        command: SLT[str],
+        callback: Callable[[Update, CCT], RT],
+        filters: BaseFilter = None,
+        allow_edited: bool = None,
+        pass_args: bool = False,
+        pass_update_queue: bool = False,
+        pass_job_queue: bool = False,
+        pass_user_data: bool = False,
+        pass_chat_data: bool = False,
+        run_async: Union[bool, DefaultValue] = DEFAULT_FALSE,
```
Comment (C-Pro) [inline bot/skills/__init__.py:59]: Why this is better than `bool = False`?

File: bot/skills/__init__.py [hunk 2]
```diff
@@ -34,16 +38,61 @@
...
+        # chat_id filtering: accept messages only from particular chat
+        chat_id_or_name = get_group_chat_id()
+        try:
+            f = Filters.chat(chat_id=int(chat_id_or_name))
+        except ValueError:
+            f = Filters.chat(username=chat_id_or_name.strip("@"))
+        filters = f if filters is None else filters & f
+
+        # run commands async by default
+        if run_async == DEFAULT_FALSE:
```
Comment (C-Pro) [inline bot/skills/__init__.py:71]: I don't get it. What does `DEFAULT_FALSE` mean if it sets `run_async` to `True`?

**Review by C-Pro — APPROVED** [review]
👍 have some comments though

File: bot/skills/__init__.py (hunk 1 above)
Comment (egregors) [reply bot/skills/__init__.py:59]: I suggest you to apply Ockhams Rasiermesser here. Generally, an answer is pretty clear: I just override parent's constructor. And `run_async: Union[bool, DefaultValue] = DEFAULT_FALSE` is just a part of it.

File: bot/skills/__init__.py (hunk 2 above)
Comment (egregors) [reply bot/skills/__init__.py:71]: Here `DEFAULT_FALSE` allow me to know, that `run_async` argument comms with _default_ value. Only in this particular case, I'd like to set it to `True`.

`DefaultValue` is a generic from `python-telegram-bot` library: `telegram.utils.helpers.DefaultValue`. They describe it like so:

```
Wrapper for immutable default arguments that allows to check, if the default value was set
    explicitly. Usage::

        DefaultOne = DefaultValue(1)
        def f(arg=DefaultOne):
            if arg is DefaultOne:
                print('`arg` is the default')
                arg = arg.value
            else:
                print('`arg` was set explicitly')
            print(f'`arg` = {str(arg)}')
...
```

In different words, I wanna change `run_async` only when nobody sets it from outside.

Comment (C-Pro) [reply bot/skills/__init__.py:71]: How this is different from setting default value to True?

Comment (egregors) [reply bot/skills/__init__.py:71]: I believe it was done to giving to parent class possibility to determinate than the `DefaultValue` was overloaded by some successors. 🤷

Comment (C-Pro) [reply bot/skills/__init__.py:71]: > I believe it was done to giving to parent class possibility to determinate than the `DefaultValue` was overloaded by some successors. 🤷 

Heresy

Comment (egregors) [reply bot/skills/__init__.py:71]: Zwar

**Pull request merged**